lib/safira_web/router.ex

```diff
@@ -4,7 +4,7 @@ defmodule SafiraWeb.Router do
 
   alias Safira.Guardian
 
-  if Mix.env == :dev do
+  if Mix.env() == :dev do
```

Comment:
What is the purpose of this change? Was it not working before

Reply:
This change is going to compare directly two atoms instead of getting string.( i guess) 
Check this:
https://elixirforum.com/t/how-to-use-if-or-other-condition-for-dev-or-prod/16235/8

Reply:
This change wasn't done by me, I just ran `mix format` against the changed files, and the formatter decided to add this. Nevertheless, @tsource98 is right, just check [this docs](https://hexdocs.pm/mix/Mix.html#env/0).